src/client/components/inputs/index.tsx

```diff
@@ -41,65 +41,47 @@ export const masks: Record<string, Mask> = {
 
 type variantType = 'light' | 'dark'
```

Comment:
If we still end up needing this, maybe convert it to an Enum and/or name it like `VariantType` to easily distinguish it between types and variables.